mirrord/cli/src/container.rs

```diff
@@ -158,6 +153,16 @@ async fn create_sidecar_intproxy(
             )
         })?;
 
+    let mut sidecar_watcher_command = Command::new(std::env::current_exe().unwrap());
```

Comment:
```suggestion
    let mut sidecar_watcher_command = Command::new(std::env::current_exe().expect("Could not get the full path of the binary (??)!"));
```

Maybe dress the error up a bit if it happens for a user (even though it's extremely unlikely)?

Reply:
I did it like the rest